### ISSUE TITLE
add deadsnakes PPA

### ISSUE THREAD

**Issue by gilesknap** [epics-containers/dev-e7#17]
Add this PPA to the container for quick access to all python versions.

**Comment by coretl** [comment]
I had to:
```
apt-get update && apt-get install software-properties-common -y && add-apt-repository -y ppa:deadsnakes/ppa && apt-get upgrade -y && \
    <install all the packages>
```

**Comment by gilesknap** [comment]
fixed in b43bc5cca54431543c303d8a364be589ef70a0d7
